src/uShip.Logging.Tests/LoggingEventPropertiesBuilderTests.cs

```diff
@@ -35,9 +35,16 @@ public void Should_sanitize_passwords()
 
             loggingProperties.Set("someInfo", "\"EmailAddress\\\":\\\"hello@meadowshotel.com\\\",\\\"Password\\\":\\\"meadows2014\\\"");
             loggingProperties.Set("moreInfo", "\"emailAddress\":\"hello@meadowshotel.com\",\"password\":\"meadows2014\"");
-
-            Assert.AreEqual(loggingProperties["someInfo"], "\"EmailAddress\\\":\\\"hello@meadowshotel.com\\\",\\\"************\\\":\\\"****\\\"");
-            Assert.AreEqual(loggingProperties["moreInfo"], "\"emailAddress\":\"hello@meadowshotel.com\",\"************\":\"****\"");
+            loggingProperties.Set("evenMoreInfo", "A potentially dangerous Request.Form value was detected from the client (_ctl0:ContentBody:conSignIn:txtExistingPassword=\"Ss9W7+&#+%@\"");
+            loggingProperties.Set("evenEvenMoreInfo", "&_ctl0%3aContentBody%3aconSignIn%3atxtExistingUserName=user1%40uship.com&_ctl0%3aContentBody%3aconSignIn%3atxtExistingPassword=shipit&_ctl0%3aContentBody%3aconSignIn%3achkSetCookie=");
```

Comment:
This essentially checks that the request body will be sanitized, right?  The `"evenEvenMoreInfo"` property appears to be formatted in the way we serialize the request body.

Reply:
The Set method calls the method that does the sanitizing using the regexes.  So all strings (RequestBody, Message, ExceptionMessage, Exception.Message).   evenEventMoreInfo is a RequestBody example.  evenMoreInfo is a Message example.